src/ansys/fluent/core/examples/downloads.py

```diff
@@ -131,3 +132,11 @@ def download_file(
 
     url = _get_file_url(filename, directory)
     return _retrieve_file(url, filename, save_path, return_only_filename)
+
+
+def path(filename: str):
+    file_path = str(Path(pyfluent.EXAMPLES_PATH) / filename)
+    if os.path.isfile(file_path):
+        return file_path
```

Comment:
I believe this could cause some inconsistencies when running tests on containers vs standalone, as `download_file` returns only a filename on container tests but a fully qualified path when standalone.

A suggestion would be:
```suggestion
def path(filename: str):
    if os.path.isabs(filename):
        return filename
    file_path = str(Path(pyfluent.EXAMPLES_PATH) / filename)
    if os.path.isfile(file_path):
        return file_path
```

Reply:
![image](https://github.com/ansys/pyfluent/assets/106588300/9431f1ec-f79b-4170-acdd-2c0174e89ca7)